src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/SearchService.kt

```diff
@@ -38,8 +39,14 @@ class SearchService(
 
   private fun searchForRecords(person: Person, personQuery: PersonQuery): List<MatchResult> {
     val highConfidenceMatches = mutableListOf<MatchResult>()
-    val matchCandidates = personRepository.findMatchCandidates(person, personQuery.query)
-    val totalElements = matchCandidates.size
+
+    val pageable = PageRequest.of(0, PAGE_SIZE)
```

Comment:
This will only take the first page of 50 Results. Will not then process other pages